index.html

```diff
@@ -429,14 +452,47 @@
 					}
 				}),
 				document.getElementById("fw_form"));
-			        if (Math.round(100 * (data.system.sketchSpaceUsedBytes / data.system.sketchSpaceTotalBytes)) >= 75)
+				if (data.hasOwnProperty("system")) // if information is unavailble this prevents issues with jquery
```

Comment:
Can you move all comments to their own line, above the code it relates to? I am not a fan of inline comments, as they usually end up hidden in the IDE and you have to scroll across to read them.

Same for all inline comments you added.